templates/include/app-form-required-info.html

```diff
@@ -0,0 +1,35 @@
+<!-- App Form Requred Info -->
+<h2>App Details - Required Info</h2>
+
+<!-- App Name -->
+<label for="id_name">Application Name</label>
+<input type="text" name="name" maxlength="255" required="" id="id_name" value="{{ application.name }}">
+
+<!-- App Client Type -->
+<label for="id_client_type">OAuth - Client Type</label>
+<select name="client_type" required="" id="id_client_type">
+	<option value="">---------</option>
+	{% if application.client_type == "public" %}
+	<option value="confidential">Confidential</option>
+	<option value="public" selected>Public</option>
+	{% else %}
+	<option value="confidential" selected>Confidential</option>
```

Comment:
these conditional blocks are the same.

Reply:
See answer for the other conditional block thing below this.

Reply:
Fixed by a new commit.